server/sonar-server/src/main/java/org/sonar/server/computation/step/ComputationSteps.java

```diff
@@ -52,11 +52,12 @@
 
       // data computation
       IntegrateIssuesStep.class,
+      CoreMetricFormulaExecutorStep.class,
       CustomMeasuresCopyStep.class,
-      ComputeFormulaMeasuresStep.class,      
 
       // SQALE measures depend on issues
       SqaleMeasuresStep.class,
+      CoverageMeasuresStep.class,
```

Comment:
Setting here this step won't work, because it needs some measures that are computed in next steps.
So you should put it just after NewCoverageAggregationStep